quick_start.adoc

```diff
@@ -1,22 +1,142 @@
 [#getting-started]
 = Getting started
 
-* <<manage-clusters,Manage clusters>>
-* <<manage-applications,Manage applications>>
-* <<manage-security,Manage security>>
-* <<observability-quick, Observe clusters>>
+* <<introduction,Introduction>>
+* <<create-and-manage-clusters,Create clusters>>
+* <<manifestwork-example, ManifestWork example>>
 
 [#introduction]
 == Introduction
 
 Coming: Architecture and intro.
 
-Before you install multicluster engine for Kubernetes, review the system configuration requirements and settings at link:../install/requirements.adoc#requirements-and-recommendations[Requirements and recommendations]. With a supported version of OpenShift Container Platform installed and running on your hub cluster, you can proceed with link:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online].
+Before you install multicluster engine for Kubernetes, review the system configuration requirements and settings at link:../requirements.adoc#requirements-and-recommendations[Requirements and recommendations]. With a supported version of OpenShift Container Platform installed and running on your multicluster engine for Kubernetes cluster, you can proceed with link:../install_connected.adoc#installing-while-connected-online[Installing while connected online].
 
+[#create-and-manage-clusters]
+== Create and manage clusters
 
-[#manage-clusters]
-== Manage clusters
+You are now ready to create and import clusters.
+From your multicluster engine for Kubernetes cluster, you can create clusters from other Kubernetes services to manage.
 
+. See link:../cluster
+_create_cli.adoc#create-a-cluster[Create a cluster] to learn about the types of managed clusters you can create.
+When you create a managed cluster, the new managed cluster imports automatically.
```

Comment:
This sentence is not true in the context of MCE, because we currently lack the cluster-deployment-import thingie.